content/blog/1-from-gantt-charts-to-generative-ai-how-agentic-ai-is-revolutionizing-project-management.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Gen AI for PM series- Part 1: From Gantt Charts to Generative AI: How
```

Comment:
Title should be sentence case.
Gen AI for PM series- Part 1: From Gantt charts to Generative AI: How Agentic AI is revolutionizing project management

But let's wait for Dale's suggestion.